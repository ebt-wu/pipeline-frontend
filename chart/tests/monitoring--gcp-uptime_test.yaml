suite: Test the GCP Pub/Sub from the Helm DXP Library
templates:
  - gcp-monitoring.yaml
values:
  - ../values.yaml
  - ../stack-values.yaml
release:
  name: extension-manager-ui-dev
  namespace: root-universe
tests:
  - it: Default Uptime Check for Extension Manager UI Site matches the Snapshot
    asserts:
      - matchSnapshot: {}
