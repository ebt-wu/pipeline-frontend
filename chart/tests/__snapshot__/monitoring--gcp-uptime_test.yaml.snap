Default Uptime Check for Extension Manager UI Site matches the Snapshot:
  1: |
    apiVersion: monitoring.cnrm.cloud.google.com/v1beta1
    kind: MonitoringUptimeCheckConfig
    metadata:
      name: extension-manager-ui-dev
      namespace: root-universe
    spec:
      displayName: HTTP Uptime Config for [extension-manager-ui-dev]
      httpCheck:
        path: /en/assets/cdm.json
        requestMethod: GET
        useSsl: true
        validateSsl: true
      monitoredResource:
        filterLabels:
          host: test-domain
          project_id: sap-gcp-dxp-test
        type: uptime_url
      period: 60s
      projectRef:
        external: sap-gcp-dxp-test
      timeout: 30s
  2: |
    apiVersion: monitoring.cnrm.cloud.google.com/v1beta1
    kind: MonitoringUptimeCheckConfig
    metadata:
      name: extension-manager-ui-dev-wc
      namespace: root-universe
    spec:
      displayName: HTTP Uptime Config for [extension-manager-ui-dev-wc]
      httpCheck:
        path: /assets/cdm.json
        requestMethod: GET
        useSsl: true
        validateSsl: true
      monitoredResource:
        filterLabels:
          host: test-domain-wc
          project_id: sap-gcp-dxp-test
        type: uptime_url
      period: 60s
      projectRef:
        external: sap-gcp-dxp-test
      timeout: 30s
